### ISSUE TITLE
sharp doesn't work with node definition

### ISSUE THREAD

**Issue by Kinzi** [directus/directus#20866]
### Describe the Bug

After updating to latest version, I get the following error on the server:

`Error: Could not load the "sharp" module using the linux-x64 runtime`

with node 18.16

I got the same for arm on my macbook but was able to fix it by changing node to 18.19.

No luck on the server though, tried 18, 19 and 20 and always get that error.

Any chance to downgrade sharp until this is fixed?

### To Reproduce

install latest version with node 18.16

### Directus Version

10.8.3

### Hosting Strategy

Self-Hosted (Custom)

**Comment by Kinzi** [comment]
```
❯ npm list sharp
contentin-cms@1.0.0 /Users/seb/Code/contentin-cms
└─┬ directus@10.8.3
  └─┬ @directus/api@15.0.0
    └── sharp@0.33.1
```

**Comment by Kinzi** [comment]
Got it working by downgrading sharp via package.json:

```
"overrides": {
    "sharp@0.33.1": "0.33.0"
  },
```

**Comment by u12206050** [comment]
For me on M1 it was fixed doing: `npm install --force @img/sharp-darwin-arm64` am guessing it might need the same on linux?

**Comment by johnny9144** [comment]
I faced the same issue on the M1 Macbook and AWS Linux server with node version 18.19.0 with Directus version 10.8.3. it was fixed by `yarn add sharp --ignore-engines` for now, but I think it is probably not a long-term solution.

**Comment by kroawen** [comment]
same issue, run `npm install --force @img/sharp-darwin-arm64`  or `yarn add sharp --ignore-engines` not fix, 
`npm WARN Could not resolve dependency:
npm WARN peer rollup@"^2.63.0" from rollup-plugin-styles@4.0.0
npm WARN node_modules/@directus/extensions-sdk/node_modules/rollup-plugin-styles
npm WARN   rollup-plugin-styles@"4.0.0" from @directus/extensions-sdk@10.3.0
npm WARN   node_modules/@directus/extensions-sdk
`

**Comment by br41nslug** [comment]
Sharp looks to have updated their minimum nodejs requirement to `18.17` (@Kinzi) https://sharp.pixelplumbing.com/install#prerequisites 

The recommended way of manually forcing OS/architecture is `npm install --cpu=arm64 --os=darwin sharp` for npm (more package manager examples on the linked page)
https://sharp.pixelplumbing.com/install#npm-v10

**Comment by Natetronn** [comment]
@Kinzi and others, I'm curious, are you getting this error when running the Docker version of Directus or is this only an issue with the NPM version, since that relies on your own version of NPM being 18.17+?

**Comment by Kinzi** [comment]
Yeah I solved it by upgrading node to 18.18

**Comment by br41nslug** [comment]
I am closing this issue for now as it seems to be solved by updating nodejs to `>18.17`. I've created an issue to track updating the docs/nodejs requirement https://github.com/directus/directus/issues/20936
